src/utils/telegram-logger.ts

```
@@ -0,0 +1,97 @@
+const threadIds = {
+  error: process.env.LOG_ERROR_THREAD_ID,
+  info: process.env.LOG_INFO_THREAD_ID,
+  warning: process.env.LOG_INFO_THREAD_ID,
+};
+
+export class TelegramLogger {
+  private static readonly botToken = process.env.TELEGRAM_BOT_TOKEN;
+  private static readonly isLocalhost = process.env.NODE_ENV === 'localhost';
+
+  static async error(message: string, context?: any, userId?: string) {
+    const formatted = this.formatLog('ERROR', message, String(context), userId);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)